### PR TITLE
UG content for Comments pane

### PR DESCRIPTION
**Pull request by KrithikaGanesan**
UG content for Comments pane

### CODE REVIEW

File: wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png:1]: Image screen shot should not contain any personal name, can you change the screen shot? use like below image
annotation comments1

File: wpf/Pdf-Viewer/Working-with-Annotations/Working-with-Annotation-Comments.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    'expand the comments pane.
+    pdfViewer.CommentSettings.IsExpanded = true
+End Sub
+
+{% endhighlight %}
+{% endtabs %}
+
+The sample project to expand the comments pane in the PDF Viewer is available in the 'GitHub'
+
+## Show/Hide the comments pane
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Working-with-Annotations/Working-with-Annotation-Comments.md:50]: Show/Hide the comments pane is not proper name can you change to Show/Hide the comments button in annotation toolbar. Change the content based on this.

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Working-with-Annotations/Working-with-Annotation-Comments.md (hunk 1 above)
Comment (KrithikaGanesan) [reply wpf/Pdf-Viewer/Working-with-Annotations/Working-with-Annotation-Comments.md]: I have changed the title and content as above

File: wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png:1]: still the author name was as "Krithika" can you change it?

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png
Comment (KrithikaGanesan) [reply wpf/Pdf-Viewer/Working-with-Annotations/Annotation-images/add-comments.png:1]: I have modified the image

**Review by Vikassekar — APPROVED** [review]

**Review by Vikassekar — APPROVED** [review]

**Pull request merged**